Proyecto/src/main/java/com/example/Proyecto/controlador/Controlador.java

```diff
@@ -16,13 +15,14 @@ public String index() {
         return "index";
     }
 
+    // http://localhost:8080/procesa
     @PostMapping("/procesa")
-    public String procesa(HttpServletRequest request, Model model ) {
-        String nombres=" Axel David García Beltrán\n Camila Alexandra Cruz Miranda 316084707\n Fernando Gerardo Flores García\nIan Israel Garcia Vazquez";
+    public String procesa(HttpServletRequest request, Model model) {
+        String nombres = " Axel David García Beltrán\n Camila Alexandra Cruz Miranda\n Fernando Gerardo Flores García\n Ian Israel Garcia Vazquez";
         String nombre = request.getParameter("nombre");
```

Comment:
I think It could be 
nombre= request.getParameter("nombre")+nombres

Reply:
Esto podemos editarlo en otro PR para que se vean reflejados los cambios. Por las indicaciones de la práctica.